core/lib/node/src/task/prometheus_exporter.rs

```diff
@@ -0,0 +1,48 @@
+use prometheus_exporter::PrometheusExporterConfig;
+use zksync_health_check::{CheckHealth, HealthStatus, HealthUpdater, ReactiveHealthCheck};
+
+use super::{IntoZkSyncTask, TaskInitError, ZkSyncTask};
+use crate::node::{NodeContext, StopReceiver};
+
+#[derive(Debug)]
+pub struct PrometheusExporterTask {
+    config: PrometheusExporterConfig,
+    prometheus_health_check: Option<ReactiveHealthCheck>,
+    prometheus_health_updater: HealthUpdater,
+}
+
+impl IntoZkSyncTask for PrometheusExporterTask {
+    type Config = PrometheusExporterConfig;
+
+    fn create(
+        _node: &NodeContext<'_>,
+        config: Self::Config,
+    ) -> Result<Box<dyn ZkSyncTask>, TaskInitError> {
+        let (prometheus_health_check, prometheus_health_updater) =
+            ReactiveHealthCheck::new("prometheus_exporter");
+
+        Ok(Box::new(Self {
+            config,
+            prometheus_health_check: Some(prometheus_health_check),
+            prometheus_health_updater,
+        }))
+    }
+}
+
+#[async_trait::async_trait]
+impl ZkSyncTask for PrometheusExporterTask {
+    fn healthcheck(&mut self) -> Option<Box<dyn CheckHealth>> {
+        self.prometheus_health_check
+            .take()
+            .map(|c| Box::new(c) as Box<dyn CheckHealth>)
+    }
+
+    async fn run(self: Box<Self>, stop_receiver: StopReceiver) -> anyhow::Result<()> {
```

Comment:
Yep, actually [it was a resource](https://github.com/matter-labs/zksync-era/blob/a283a46c642ad4893ca93bdfea5578bd4d1a9291/core/lib/node/src/resource/stop_receiver.rs), but I decided against it, since I thought that it'd be really easy to forget to request it and create a task that hangs. It also was clunky, since this resource was the only one that [is implicitly added](https://github.com/matter-labs/zksync-era/blob/a283a46c642ad4893ca93bdfea5578bd4d1a9291/core/lib/node/src/node/mod.rs#L135) and I wanted to avoid it.